### PR TITLE
creating handshake accepted notification TM-2078

### PR DESCRIPTION
**Pull request by SidelineCory24**

### CODE REVIEW

**Comment by SidelineCory24** [comment]
Still need to trigger the notification only under the CDO

**Comment by SidelineCory24** [comment]
Notification triggers whenever `Submit Bid` is clicked on the `Bid Tracker`.
Current wording on the notification is based off of the SS in the ticket.

**Comment by elizabeth-jimenez** [comment]
I dont understand why `WrappedComponent` did not work when we tried it earlier 🤷‍♀️ 

feel free to add in a new test for your new `mapDispatchToProps` function

File: src/Components/BidTracker/OverlayAlert/DraftAlert/DraftAlert.jsx [hunk 1]
```diff
@@ -81,6 +84,15 @@ DraftAlert.contextTypes = {
...
+  handshakeAccepted: PropTypes.func,
 };
 
-export default DraftAlert;
+DraftAlert.defaultProps = {
+  handshakeAccepted: EMPTY_FUNCTION,
+};
+
+export const mapDispatchToProps = dispatch => ({
+  handshakeAccepted: () => dispatch(handshakeAccepted()),
```
Comment (elizabeth-jimenez) [inline src/Components/BidTracker/OverlayAlert/DraftAlert/DraftAlert.jsx:95]: we did this earlier, but reverted our changes lol give the props var a different name than the actions'

File: src/Components/BidTracker/OverlayAlert/DraftAlert/DraftAlert.jsx [hunk 2]
```diff
@@ -2,14 +2,17 @@ import { Component } from 'react';
 import StaticDevContent from 'Components/StaticDevContent';
 import PropTypes from 'prop-types';
 import FontAwesome from 'react-fontawesome';
-import { BID_OBJECT } from '../../../../Constants/PropTypes';
+import { connect } from 'react-redux';
+import { BID_OBJECT, EMPTY_FUNCTION } from '../../../../Constants/PropTypes';
```
Comment (elizabeth-jimenez) [inline src/Components/BidTracker/OverlayAlert/DraftAlert/DraftAlert.jsx:6]: can probably be shortened to 'Constants/PropTypes'

File: src/sass/_toast.scss [hunk 3]
```diff
@@ -145,4 +145,8 @@
 
 .Toastify__toast-container { // scss-lint:disable SelectorFormat
   width: 400px;
+    .Toastify__toast--dark {
```
Comment (elizabeth-jimenez) [inline src/sass/_toast.scss:148]: 👍

File: src/Components/BidTracker/Messages/handshakeAccepted.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+const HandshakeAccepted = ({ name, position, bid }) => (
+  <span>
+    {name}, has accepted a handshake for position:
+    <Link to={position.link}>{position.name}</Link>
+    <div className="bottom-section">
+      Go to Clients Bid Tracker to approve and register the Handshake
```
Comment (elizabeth-jimenez) [inline src/Components/BidTracker/Messages/handshakeAccepted.jsx:9]: "Go to Client's Bid Tracker to register the Handshake" we don't really have an approve step. This saves us space too!

File: src/Components/BidTracker/OverlayAlert/DraftAlert/DraftAlert.jsx [hunk 5]
```diff
@@ -2,14 +2,17 @@ import { Component } from 'react';
...
 import { NO_POST, NO_SKILL, NO_GRADE } from '../../../../Constants/SystemMessages';
 import { getPostName, formatDate } from '../../../../utilities';
+import { handshakeAccepted } from '../../../../actions/bidTracker';
```
Comment (elizabeth-jimenez) [inline src/Components/BidTracker/OverlayAlert/DraftAlert/DraftAlert.jsx:9]: i think you can shorten all of these:
```
import { BID_OBJECT, EMPTY_FUNCTION } from 'Constants/PropTypes';
import { NO_POST, NO_SKILL, NO_GRADE } from 'Constants/SystemMessages';
import { getPostName, formatDate } from 'utilities';
import { handshakeAccepted } from 'actions/bidTracker';
```

File: src/Components/BidTracker/OverlayAlert/DraftAlert/DraftAlert.test.jsx [hunk 6]
```diff
@@ -31,16 +33,20 @@ describe('DraftAlertComponent', () => {
...
-      <DraftAlert {...props} />,
+      <DraftAlert.WrappedComponent {...props} />,
     );
     expect(toJSON(wrapper)).toMatchSnapshot();
   });
 });
+
+describe('mapDispatchToProps', () => {
+  testDispatchFunctions(mapDispatchToProps);
+});
```
Comment (elizabeth-jimenez) [inline src/Components/BidTracker/OverlayAlert/DraftAlert/DraftAlert.test.jsx:52]: 😻

File: src/reducers/bidTracker/bidTracker.js [hunk 7]
```diff
@@ -5,4 +5,13 @@ export function handshakeOfferedNotification(state = { name: null, position_name
     default:
       return state;
   }
-}
\ No newline at end of file
+}
+
+export function handshakeAcceptedNotification(state = { name: null, position_name: null, bidtracker: null }, action) {
```
Comment (elizabeth-jimenez) [inline src/reducers/bidTracker/bidTracker.js:10]: We'll prolly have to update this later 🤪  me too with the `handshakeOfferedNotification` above. Just realized that I updated to `name, position, bid` but didn't update in here. Not worth the effort to change now, but just something to keep in mind if we happen upon an error, in the future, when we hook these up.

**Comment by elizabeth-jimenez** [comment]
Looks good to go except a few small comments and making sure the HS Offered toast is not affected by the new styling 🦜

**Comment by elizabeth-jimenez** [comment]
image
it's a beauty 😻

File: src/Components/BidTracker/Messages/handshakeAccepted.jsx [hunk 8]
```diff
@@ -6,7 +6,7 @@ const HandshakeAccepted = ({ name, position, bid }) => (
     {name}, has accepted a handshake for position:
     <Link to={position.link}>{position.name}</Link>
     <div className="bottom-section">
-      Go to Clients Bid Tracker to approve and register the Handshake
+      Go to Client&apos;s Bid Tracker to register the Handshake
```
Comment (elizabeth-jimenez) [inline src/Components/BidTracker/Messages/handshakeAccepted.jsx:9]:
```
      {/* eslint-disable-next-line react/no-unescaped-entities */}
      Go to Client's Bid Tracker to register the Handshake
```
this should work 😫 ESLint!

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 🧁 🥮 👍

File: src/Components/BidTracker/Messages/handshakeAccepted.jsx (hunk 8 above)
Comment (SidelineCory24) [reply src/Components/BidTracker/Messages/handshakeAccepted.jsx]: okay cool, didn't know which way to go lol

**Comment by SidelineCory24** [comment]
Testing github account changes

**Pull request merged**